src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -970,8 +971,9 @@ private void startContainer() {
                 }
             }
 
+            info("Starting Docker image...");
```

Comment:
Should be `"Starting Docker container..."`

Reply:
Sure